### PR TITLE
Add module name opt to lake wrapper

### PR DESCRIPTION
**Pull request by kavyasreedhar**
adds a -n option (default: LakeWrapper) to set the name of the wrapper module generated for compiler team

### CODE REVIEW

File: lake/utils/util.py [hunk 1]
```diff
@@ -161,10 +165,10 @@ def generate_lake_config_wrapper(configs_list, configs_file, lake_file):
                 if add == 1:
                     not_configs.append(line)
 
-    with open("LakeWrapper.v", "w+") as wrapper:
+    with open(f"LakeWrapper_{module_name}.v", "w+") as wrapper:
 
         # write top level interface (without config regs)
-        wrapper.write("module LakeWrapper (\n")
+        wrapper.write("module " + module_name + " (\n")
```
Comment (mbstrange2) [inline lake/utils/util.py:171]: use f string here?

Comment (mbstrange2) [inline lake/utils/util.py:171]: format string?
```suggestion
        wrapper.write(f"module {module_name} (\n")
```

File: tests/wrapper_lake.py [hunk 2]
```diff
@@ -44,15 +46,15 @@ def error(usage):
 def main(argv):
     usage = "File usage: python wrapper.py [-c / --csv_file] [csv_file path relative to LAKE_CONTROLLERS environment variable]"
     usage += " [-s / --stencil_valid] [True or False indicating whether or not to generate hardware with stencil_valid (default: True)"
+    usage += " [-n] [module name for LakeWrapper module (default: LakeWrapper)]"
     try:
-        options, remainder = getopt.getopt(argv, 'c:s:', ["csv_file=", "stencil_valid="])
+        options, remainder = getopt.getopt(argv, 'c:s:n:', ["csv_file=", "stencil_valid="])
```
Comment (mbstrange2) [inline tests/wrapper_lake.py:51]: Maybe try using ArgParser?

https://github.com/StanfordAHA/garnet/blob/master/tests/test_memory_core/test_memory_core.py#L1170

**Review by mbstrange2 — CHANGES_REQUESTED** [review]
Small formatting stuff, otherwise LGTM

File: tests/wrapper_lake.py (hunk 2 above)
Comment (kavyasreedhar) [reply tests/wrapper_lake.py:51]: sounds good, just pushed.

**Review by mbstrange2 — APPROVED** [review]
LGTM

**Pull request merged**